docs/DeveloperGuide.md

```diff
@@ -146,6 +146,8 @@ The `View` abstract class provides a common interface for rendering and navigati
 - `printView(TaskManager taskManager)`: This method is responsible for rendering the view to the console based on the provided `TaskManager` instance.
 - `next()`: This method advances the view to the next period (e.g., next week or next month).
 - `previous()`: This method moves the view to the previous period (e.g., previous week or previous month).
+- 
+![Time.png](images%2Fclass%2FTime.png)
```

Comment:
Image path likely will not be found causing the images to not load. Possibly rectify with slashes `/` instead of `%2F`